static_report/src/App.tsx

```diff
@@ -7,6 +7,29 @@ import { useHashLocation } from './hooks/useHashLcocation';
 import { SingleReportList } from './components/SingleReport/SRList';
 import { ComparisonReportList } from './components/ComparisonReport/CRList';
 
+import * as Sentry from '@sentry/browser';
+import { BrowserTracing } from '@sentry/tracing';
+
+const sentry_dns = window.PIPERIDER_METADATA.sentry_dns || null;
+if (sentry_dns) {
+  const sentry_env = window.PIPERIDER_METADATA.sentry_env || 'development';
+  const piperider_version = window.PIPERIDER_METADATA.version || 'unknown';
+  const release_version =
+    sentry_env === 'development' ? null : piperider_version;
```

Comment:
Perhaps these default values should be sourced via `dotenv`?
and stored as `.env.development` globally for both CLI and FE?